csv/result.go

```diff
@@ -397,6 +397,8 @@ func readMetadata(r *csv.Reader, c ResultDecoderConfig, extraLine []string) (tab
 				return tableMetadata{}, errors.Wrapf(err, "column %q has invalid default value", label)
 			}
 			defaultValues[j] = v
+		} else {
+			defaultValues[j] = values.NewNull(flux.SemanticType(cols[j].ColMeta.Type))
```

Comment:
Can you add a constant for the `nullValue` and set it to `""` for now? We want to make it clear how the null value interacts with the default value. The if statement above it would become switch statement with three cases:

```go
switch defaults[j] {
    case nullValue:
        // Set explicit null as default
        defaultValues[j] = values.NewNull(flux.SemanticType(cols[j].ColMeta.Type))
    case "":
        // Do nothing there is no default
    default:
        v, err := decodeValue(defaults[j], cols[j])
        // ...
}
```
Technically the second case will not be possible until `nullValue` is dynamic. But it makes it clear that its still possible to not have a default value.

Reply:
I can't write the code like you've done it here because Go complains about duplicate cases, but I will add `nullValue` and comments to this effect.